rpgame/src/main/java/pk/rpgame/view/ExplorationView.java

```diff
@@ -1,22 +1,61 @@
 package pk.rpgame.view;
 
 import pk.rpgame.model.Room;
-
 import java.util.List;
+import java.util.Scanner;
+
+interface MenuClickListener {
+    void onActionClick(int num);
+}
+
+enum PickUpItems {
+    PICK_UP,
+    DONT_PICK_UP
+}
 
 public class ExplorationView {
-    private final Room currentRoom;
 
-    public ExplorationView(Room currentRoom) {
-        this.currentRoom = currentRoom;
-    }
+    private MenuClickListener listener;
 
-    public void printRoomDescription() {
+    public void printRoomDescription(String roomName) {
+        System.out.println("Znajdujesz się w pomieszczeniu " + roomName);
         System.out.println("W tym pokoju jesteś bezpieczny. Nie ma tu żadnych potworów.");
     }
 
-    public void printNearestRooms() {
-        List<Room> nearestRooms = this.currentRoom.getNearestRooms();
+    public void setListener(MenuClickListener listener) {
+        this.listener = listener;
+    }
+
+    private int getAction() {
+        Scanner keyboard = new Scanner(System.in);
+        return keyboard.nextInt();
+    }
+
+    public void printNearestRooms(List<Room> nearestRooms) {
         System.out.println("Możliwe pomieszczenia, do których możesz się udać: " + nearestRooms);
     }
+
+    public PickUpItems pickUpItems() {
+        System.out.println("Możesz podnieść znalezione przedmioty. Wybierz '1' aby je podnieść lub '0' aby nic nie robić.");
+        System.out.println("Podniesienie pancerza lub broni spowoduje upuszczenie aktualnie noszonych przedmiotów z tej kategorii");
+        int action = getAction();
+        while (true) {
+            if (action == 1) {
+                return PickUpItems.PICK_UP;
+            } else if (action == 0) {
+                return PickUpItems.DONT_PICK_UP;
+            } else {
+                System.out.println("Wybierz właściwą cyfrę!");
+            }
+        }
+    }
+
+    public void showMenu() {
+        System.out.println("1. Eksploruj pomieszczenie");
+        System.out.println("2. Przejdź do następnego pokoju");
+        System.out.println("3. Pokaż mapę");
+        System.out.println("4. Pokaż menu");
+        int action = getAction();
+        listener.onActionClick(action);
+    }
 }
```

Comment:
Planujesz gdzieś dodać metody wypisujące listę pokoi i potworów pokoju?
W sumie nie wiem czy spoko byłoby zrobić np. metod/y pobierającej listę Room i LivingEntity 
-> wypisującej ich ponumerowaną listę 
-> pytającą o wybór (np. przy wyborze potowa do walki) 
-> i zwracającą obiekt wybranego potwora czy itemu
Choć do wyboru itemu dałeś opcję wybrania wszystkich itemów lub żadnego, więc samo wylistowanie itemów by wystarczyło w tym przypadku. Co sądzisz?

Reply:
Wybór potwora do walki raczej będzie w klasie FightView. 
Nie jestem przekonany co do wypisywania listy pokoi, skoro mamy wyświetlanie mapy, bo to robi to samo, tylko ładniej graficznie. 
Co do itemków, założyłem, że będzie max 1 item na pokój, bo nie mamy ich dużej różnorodności, plus tak jest prościej xd